src/main/java/com/chat/yourway/service/interfaces/ContactService.java

```diff
@@ -0,0 +1,11 @@
+package com.chat.yourway.service.interfaces;
+
+import com.chat.yourway.dto.request.ContactRequestDto;
+import com.chat.yourway.model.Contact;
+
+public interface ContactService {
```

Comment:
General comment - all services should be covered at-least by unit tests, ideally by integrational as well. You can find examples at another active PR

Reply:
This is the task of Dmitry Telyukov. If it is needed, I will try to do it.